tests/ports/psoc6/board_only_hw/single/pdm_pcm.py.exp

```diff
@@ -0,0 +1 @@
+machine.PDM_PCM: PLL0 freq is changed from 48000000 to 24576000. This will affect all resources clock freq sourced by PLL0.
```

Comment:
Yes and right now failures in i2s in because of same reason.
1. Maybe in msg we can mention only what is set to and not what was the existing frequency. (Easy way)
2. We could have clock functions exposed in MPY so that from application side set back to whatever was the initial frequency before starting the test.
3. In deinit() implementation of PDM, I2C etc, set back clocks to reset value.

Reply:
Good points. Let´s discuss it in our daily :)

Reply:
As discussed, msg is fixed now and also ticket create in backlog.